src/facade/dragonfly_listener.cc

```diff
@@ -429,8 +429,12 @@ DispatchTracker::DispatchTracker(absl::Span<facade::Listener* const> listeners,
 }
 
 void DispatchTracker::TrackOnThread() {
-  for (auto* listener : listeners_)
+  for (auto* listener : listeners_) {
+    if (listener->IsPrivilegedInterface()) {
```

Comment:
I think we  shouldn't check this in the dispatch tracker. It's better to provide listeners without admin port